buildout.cfg

```diff
@@ -1,20 +1,34 @@
 [buildout]
-index = http://pypi.bluedynamics.eu
-parts = instance
-extends = http://dist.plone.org/release/4.2-latest/versions.cfg
-unzip = true
-develop = .
+extends =
+    https://raw.github.com/collective/buildout.plonetest/master/test-4.3.x.cfg
+    https://raw.github.com/collective/buildout.plonetest/master/qa.cfg
 
-[instance]
-recipe = plone.recipe.zope2instance
-http-address = 8080
-user = admin:admin
-eggs =
-    Pillow
-    plone.reload
-    plone.app.debugtoolbar
-    collective.folderorder
+package-name = collective.folderorder
+package-extras = [test]
 
-zcml =
-    plone.reload
-    collective.folderorder
+parts +=
+    createcoverage
+    i18ndude
+    omelette
+    rebuild_i18n-sh
+
+[code-analysis]
+recipe = plone.recipe.codeanalysis[recommended]
+directory = ${buildout:directory}/src/collective/folderorder
+clean-lines = True
+multiprocessing = True
+pre-commit-hook = True
+return-status-codes = False
+flake8-ignore = E501,P001,T000
+
+[i18ndude]
+recipe = zc.recipe.egg
+eggs = i18ndude
+
+[omelette]
```

Comment:
Actually, would it be a good idea to add this part do `qa.cfg` in `buildout.plonetest` as well, or is it out of the scope of `Quality Assurance` definition?

Reply:
let's have a discussion around tis in other place.